### ISSUE TITLE
[2.0.0 RC1] Tabs - mb3 BUG

### ISSUE THREAD

**Issue by joomleb** [JoomShaper/helix-ultimate#178]
Hi guys,
PHP 7.4.20 + Joomla 3.9.28 + Helix Ultimate 1.1.4 and confirmed on 2.0.0 RC1

As you can check [on this page as example](https://test3.egulp.net/es/turismo/alojamiento/lychee?room_quantity=1), the file:
../templates/shaper_helixultimate/html/layouts/libraries/cms/html/bootstrap/**starttabset.php**
is adding a "_joomla-tabs nav nav-tabs mb-3_" classes

So, a Bootstrap **mb-3** margin bottom is applied creating an incomplete border at the top of the Tabs - I add to images for you.
To have a margin bottom on start tabset have no any sense and will always create a layout bug because for sure all the Tabs have borders, so,
![Property-page-Tabs-mb3-issue](https://user-images.githubusercontent.com/14607633/124647278-5e467c80-de5b-11eb-8f3d-814dae43d648.png)

**Solution**:
- to delete the mb3
- OR to change mb3 with **mt3** (maybe m**b**3 was just a typing error)

**Comment by mdamjadrana** [comment]
If possible Add "mb-0" Class or make a custom css .

**Comment by joomleb** [comment]
Hi @mdamjadrana 
Sorry, I don't understand your answer.
I reported it because the mb3 class is breaking all the Standards Bootstrap Tabs borders.

Please, Am I missing anything ?
